sparsity/sparse_frame.py

```diff
@@ -50,6 +50,10 @@ def _append_zero_row(csr):
     )
 
 
+def _batch_get_loc(index: pd.Index, keys: list):
+    return [index.get_loc(key) for key in keys]
```

Comment:
Where is this used? I think there is a pandas function which does this.

Reply:
Oh, you're right, it's not used anymore.